deployment/mesh-infra/storage/minio/kustomization.yaml

```diff
@@ -1,5 +1,50 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+# NOTE (1), (2)
 resources:
-- base.yaml
+# - base.yaml
```

Comment:
Note down that we keep this as an example of a very simple MinIO setup lifted and tweaked from the DirectPV docs. Tweaks include Reloader and own K8s secret---see `mesh-infra/security/secrets`.

Reply:
actually I zapped `base.yaml` since the latest MinIO uses a different setup---see their example deployment. Also removed the MinIO secret that the deployment was referencing.